app/views/devise/sessions/new.html.erb

```diff
@@ -2,6 +2,7 @@
 
 <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
   <div class="form-inputs">
+    <%= f.input :name, required: false, autofocus: true %>
     <%= f.input :email, required: false, autofocus: true %>
     <%= f.input :password, required: false %>
```

Comment:
`autofocus` attribute is only used on one input per form. It will focus the cursor on that input when the page loads. Choose one of the fields (line 5 or 6) to remove autofocus attribute from.